app/assets/javascripts/schedule_dropdown.js

```diff
@@ -2,8 +2,14 @@ var day_picker_box = document.getElementById("day_picker_box");
 var element_to_show = document.getElementsByClassName("day_picker__text");
 
 day_picker_box.addEventListener("click", function(){
-  alert("You Clicked Me!");
   for (var i = element_to_show.length - 1; i >= 0; i--) {
     element_to_show[i].classList.remove("day_picker__text--hide")
+    element_to_show[i].addEventListener("click", function(){
+      alert(event.target.innerHTML)
```

Comment:
Here you are referencing an `event` that hasn't been passed to the EventListener function.